### ISSUE TITLE
Bug: Create Proposal form has a lot of bugs in it

### ISSUE THREAD

**Issue by 0xZakk** [0xPrimordia/impact-stream-app#243]
The form for creating a proposal has a lot of bugs in it that lead to being bounced around between screens and submitting the form before it's complete.

https://github.com/0xPrimordia/impact-stream-app/assets/4991553/5cc3c36c-a205-4a98-b8c1-fb1ce5406c07

**Comment by kevincompton** [comment]
This is due to focus order prioritizing the back button above other elements. For desktop, the focus should be more clearly outlined and the focus order should prioritize following form elements, then next button and back button last.

**Comment by 0xZakk** [comment]
Next step: Test and try to reproduce on mobile device

**Comment by thelostone-mc** [comment]
It works alright on mobile The milestone section was broken but now is fixed
